packages/interactivity-router/src/index.ts

```diff
@@ -403,30 +397,28 @@ function a11ySpeak( messageKey: keyof typeof navigationTexts ) {
 			} catch {}
 		} else {
 			// Fallback to localized strings from Interactivity API state.
+			// @todo This block is for Core < 6.7.0. Remove when support is dropped.
+
+			// @ts-expect-error
 			if ( state.navigation.texts?.loading ) {
```

Comment:
Just confirming, does this state come from the server in WP 6.6 and 6.5? Was it removed in WP core for WP 6.7?

Reply:
https://github.com/WordPress/wordpress-develop/pull/7304 is expected to remove it in 6.7 and switch to the script module data passing. It was in the Interactivity API store state before that change, so in WordPress 6.6 it was in the Interactivity API store state.

Reply:
Excellent. Thank you for confirmation.

Reply:
I may not have answered that fully. [Yes, it's server provided state in WordPress 6.6.](https://github.com/WordPress/wordpress-develop/blob/b071c28f8b6be06e464ccf582877e66c02859679/src/wp-includes/interactivity-api/class-wp-interactivity-api.php#L1019-L1030)